docs/odcs.rst

```diff
@@ -0,0 +1,123 @@
+ODCS Integration
+================
+
+When a container build is started, the user usually provides the URL to a yum
+repo that contains the packages to be installed when building the container
+image. Usually, this is a puddle containing signed packages.
+
+In the case a yum repo URL is not provided, and Koji integration is enabled,
+OSBS falls back to the yum repo associated with the build tag for the Koji build
+target (required parameter for Koji container builds). Packages in this yum repo
+are not always signed. This is not ideal as it’s trivial to produce a container
+image that should never be shipped.
+
+Another problem with the current state is that to produce an image with signed
+packages, a parameter is needed (yum repo URL). To complicate matters, this yum
+repo URL will likely be different for future builds. This poses a huge problem
+for OSBS autorebuilds. In this case, OSBS will automatically rebuild layered
+images upon detected changes to parent image. It is necessary that these
+rebuilds pick up the latest, most relevant, signed packages.
+
+The On Demand Compose Service (`odcs`_) project has been created to address
+current limitations in obtaining a yum repo with signed packages. The general
+idea is to not require the user to provide a yum repo url when building
+container images.  Instead, OSBS will use ODCS to generate a yum repo with the
+relevant signed packages based on user configuration.
+
+API Changes
+-----------
+
+The following parameters will be used by ``buildContainer`` task defined by
+``koji-containerbuild`` plugin:
+
+``odcs_signed <bool, default True>`` When set to False, OSBS will provide an
+empty list of ``sigkeys`` to ODCS. An empty list of ``sigkeys`` indicates the
+signature of packages will not be checked causing the compose to potentially
+include unsigned packages. If ODCS integration is not in place, this parameter
+has no effect.
+
+``container.yaml`` Changes
+--------------------------
+
+In git repo, alongside Dockerfile, the ``container.yaml`` file will be enhanced
+to contain a "compose" section. This section will include all the required
+information for requesting a compose to be created by ODCS::
+
+    compose:
+      # Optional, default False.
+      inherit: true
```

Comment:
`inherit` is nice and short, but it feels not self-explanatory enough, would `inherit_from_parent` work?

Reply:
I think `inherit_from_parent` is a bit too long and more error prone. I clarified the comment above to say: 
>  Optional, default False. Inherit compose used for building parent image